bool is_culbuto_touched
---
pobax_playground/SoundTrajectory baxter_sound_trajectory